### DISCUSSION TITLE
Is there a way to have long press bindings?

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [hyprwm/Hyprland#424] [Q&A]
![image](https://user-images.githubusercontent.com/68589851/180652635-60c5a13b-b556-436b-9713-e10451e62b6e.png)
Currently, this example from the wiki, does resize the window, but I have to spam the keys... which isn't ideal.
I hope that I can long press a button, and then it will keep resizing

**Comment by vaxerski** [comment]
not yet, will be implemented though, it is on the bucket list.
